workers/src/main/java/edu/cornell/testoutputstream/TestResultsRecord.java

```diff
@@ -0,0 +1,21 @@
+package edu.cornell.testoutputstream;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
+@AllArgsConstructor
+public class TestResultsRecord {
```

Comment:
Can be changed to a record

```suggestion
public record TestResultsRecord {
```